__tests__/app.test.js

```diff
@@ -0,0 +1,31 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe.only("/api/topics", () => {
+  describe("GET", () => {
+    test("200: Responds with array of topic objects with the properties of slug and description", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          expect(Array.isArray(body.topics)).toBe(true);
+          expect(body.topics.length > 0).toBe(true);
```

Comment:
i personally would opt to check for the actual length of our test categories here (seeing as it's a number we know and can see pretty easily has an expected length of 4 items), but this is fine too as it does check we are getting something back

Reply:
Thank you Hev!!